### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by lbdreyer**

### CODE REVIEW

File: nc_time_axis/tests/integration/test_plot.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        # If a plotting test bombs out it can leave the current figure
+        # in an odd state, so we make sure it's been disposed of.
+        plt.close('all')
+
+    def Test_360_day_calendar(self):
+        dates = [nc_time_axis.CalendarDateTime(netcdftime.datetime(
+            1986, month, 30), '360_day') for month in range(1, 6)]
+        line1, = plt.plot(dates)
+        result_ydata = line1.get_ydata()
+        self.assertArrayEqual(expected_ydata, result_ydata)
```
Comment (bjlittle) [inline nc_time_axis/tests/integration/test_plot.py]: @lbdreyer And the `expected_ydata` is ...

Comment (pelson) [inline nc_time_axis/tests/integration/test_plot.py]: Untested apparently? Problems with travis-ci?

Comment (bjlittle) [inline nc_time_axis/tests/integration/test_plot.py]: @lbdreyer Can you test this in your conda environment with the appropriate `netcdf` version.

Also make sure `python setup.py test` is discovering and running this integration test ...

Comment (pelson) [inline nc_time_axis/tests/integration/test_plot.py]: I don't know if the capitalised T is important...

Comment (lbdreyer) [inline nc_time_axis/tests/integration/test_plot.py]: IT WAS THE CAPITALISED T!!

Thank you, @pelson. The test not running has been flumoxing me since I wrote this about a week ago!

**Comment by lbdreyer** [comment]
I have rebased so that this includes the changes in #12 which makes use of the updated netcdf4 version.

File: nc_time_axis/tests/integration/test_plot.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    def test_360_day_calendar(self):
+        datetimes = [netcdftime.datetime(1986, month, 30)
+                     for month in range(1, 6)]
+        cal_datetimes = [nc_time_axis.CalendarDateTime(dt, '360_day')
+                         for dt in datetimes]
+        line1, = plt.plot(cal_datetimes)
+        result_ydata = line1.get_ydata()
+        result_datetimes = [cdt.datetime for cdt in result_ydata]
+        np.testing.assert_array_equal(result_datetimes, datetimes)
+        self.assertIs('360_day', result_ydata[0].calendar)
```
Comment (lbdreyer) [inline nc_time_axis/tests/integration/test_plot.py]: I suppose I could add a `__eq__` method to CalendarDateTime to save from having to always test the equality of the datetimes, and the calendars separately (which I have also had to do in a [test that was included in PR#12](https://github.com/lbdreyer/nc-time-axis/blob/1ff3e091c34052f0187756d64cd420af58cb677d/nc_time_axis/tests/unit/test_NetCDFTimeConverter.py#L21)).

Comment (bjlittle) [inline nc_time_axis/tests/integration/test_plot.py]: @lbdreyer Seems like a good idea to me, but I'd do it in a separate PR from #12

**Comment by lbdreyer** [comment]
@bjlittle I have now fixed the pep8 issue with the integration test
